homeassistant/components/slack/sensor.py

```diff
@@ -0,0 +1,53 @@
+"""Slack platform for sensor component."""
+from __future__ import annotations
+
+from slack import WebClient
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+import homeassistant.util.dt as dt_util
+
+from . import SlackEntity
+from .const import ATTR_SNOOZE, DOMAIN, SLACK_DATA
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Slack select."""
+    async_add_entities(
+        [
+            SlackSensorEntity(
+                hass.data[DOMAIN][entry.entry_id][SLACK_DATA],
+                SensorEntityDescription(
+                    key="do_not_disturb_until",
```

Comment:
If the description is static we can define it at the module level as a constant.